Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -245,40 +246,55 @@ protected function requireConstraintsImposedByAncestorsAreMet(
         array $parentNodeAggregateIds,
         ContentRepository $contentRepository
     ): void {
-        foreach ($parentNodeAggregateIds as $parentNodeAggregateId) {
-            $parentAggregate = $this->requireProjectedNodeAggregate(
-                $contentStreamId,
-                $parentNodeAggregateId,
-                $contentRepository
-            );
-            try {
-                $parentsNodeType = $this->requireNodeType($parentAggregate->nodeTypeName);
-                $this->requireNodeTypeConstraintsImposedByParentToBeMet($parentsNodeType, $nodeName, $nodeType);
-            } catch (NodeTypeNotFound $e) {
-                // skip constraint check; Once the parent is changed to be of an available type,
-                // the constraint checks are executed again. See handleChangeNodeAggregateType
-            }
+        /** @var array<string, bool> $parentNodeAggregateClassificationsById */
+        $parentNodeAggregateClassificationsById = [];
 
+        foreach ($parentNodeAggregateIds as $parentNodeAggregateId) {
             foreach (
                 $contentRepository->getContentGraph()->findParentNodeAggregates(
                     $contentStreamId,
                     $parentNodeAggregateId
                 ) as $grandParentNodeAggregate
             ) {
-                /* @var $grandParentNodeAggregate NodeAggregate */
+                $parentAggregate = $this->requireProjectedNodeAggregate(
+                    $contentStreamId,
+                    $parentNodeAggregateId,
+                    $contentRepository
+                );
                 try {
                     $grandParentsNodeType = $this->requireNodeType($grandParentNodeAggregate->nodeTypeName);
+                    $parentNodeAggregateClassificationsById[$parentNodeAggregateId->value] = $parentAggregate->classification;
                     $this->requireNodeTypeConstraintsImposedByGrandparentToBeMet(
                         $grandParentsNodeType,
                         $parentAggregate->nodeName,
                         $nodeType
                     );
                 } catch (NodeTypeNotFound $e) {
-                    // skip constraint check; Once the grand parent is changed to be of an available type,
+                    // skip constraint check; Once the grandparent is changed to be of an available type,
                     // the constraint checks are executed again. See handleChangeNodeAggregateType
                 }
             }
         }
+
+        if (!in_array(NodeAggregateClassification::CLASSIFICATION_REGULAR, $parentNodeAggregateClassificationsById, true)) {
+            // all parent node aggregates are tethered, so we don't run the direct constraint checks
+            return;
+        }
```

Comment:
im not sure what should happen, if not all parent aggregates are tethered?
Here i decided to still run the grand parent constraints in this case.